File: C/unit_3/Ff.c
#include<stdio.h>
int main()
{
  int r,s,a;
  printf("Enter The Age of Ram : \n");
  scanf("%d",&r);
  printf("Enter The Age of Shyam : \n");
  scanf("%d",&s);
  printf("Enter the Age of Ajay : \n");
  scanf("%d",&a);
  if(a<r){
      if(a<s)
        printf("Ajay is Youngest !!!");
      else
        printf("Shyam is Youngest !!!");
  }
  else if(r<a){
      if(s>r)
        printf("Ram is Youngest !!!");
      else
        printf("Shyam is Youngest !!!");
  }
  else
    printf("Shyam is Youngest !!!");
  return 0;
}

File: Java/unit_3/Ff.java
package Java.unit_3;

import java.util.Scanner;

public class Ff {
    public static void main(String[] args) {
        int r, s, a;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Age of Ram : \n");
        r = sc.nextInt();
        System.out.println("Enter the Age of Shyam : \n");
        s = sc.nextInt();
        System.out.println("Enter the Age of Ajay : \n");
        a = sc.nextInt();
        if (a < r) {
            if (a < s)
                System.out.println("Ajay is the Youngest Boy !!! ");
            else
                System.out.println("Shyam is the Youngest Boy !!! ");
        } else if (r < a) {
            if (s > r)
                System.out.println("Ram is the Youngest Boy !!! ");
            else
                System.out.println("Shyam is the Youngest Boy !!! ");
        } else {
            System.out.println("Shyam is the Youngest Boy !!! ");
        }
        sc.close();
    }
}
